util/cron/common.bash

```diff
@@ -13,13 +13,20 @@ if [ -z "${CHPL_SOURCED_BASHRC}" -a -f ~/.bashrc ] ; then
     export CHPL_SOURCED_BASHRC=true
 fi
 
+if [ "${USE_LLVM_13}" == "true" ] ; then
+  LLVM_VERSION=13
+else
+  # blank for default
+  LLVM_VERSION=
+fi
+
```

Comment:
I have a slight preference for keeping this isolated to the gpu scripts. Maybe something like:


```sh
# use older llvm until https://github.com/chapel-lang/chapel/issues/19740 is resolved
source /cray/css/users/chapelu/setup_system_llvm.bash 13
export OFFICIAL_SYSTEM_LLVM=true # lie to prevent common.bash adding a newer llvm
```